src/main.ts

```diff
@@ -14,69 +15,140 @@ const COMMANDLINE_TOOLS_LIN_URL = `https://dl.google.com/android/repository/comm
 const HOME = os.homedir()
 const ANDROID_HOME_DIR = path.join(HOME, '.android')
 const ANDROID_HOME_SDK_DIR = path.join(ANDROID_HOME_DIR, 'sdk')
-const ANDROID_REPOSITORIES_CFG = path.join(ANDROID_HOME_DIR, 'repositories.cfg')
+const ANDROID_SDK_ROOT = process.env['ANDROID_SDK_ROOT'] || ANDROID_HOME_SDK_DIR
```

Comment:
II believe this is incompatible with current ubuntu builders, since the building process has no rights to write to the directory where the sdk is located